drivers/gpu/drm/amd/amdgpu/psp_v12_0.c

```diff
@@ -49,9 +49,14 @@ static int psp_v12_0_init_microcode(struct psp_context *psp)
 {
 	struct amdgpu_device *adev = psp->adev;
 	const char *chip_name;
+#ifdef __linux__
 	char fw_name[30];
 	int err = 0;
 	const struct ta_firmware_header_v1_0 *ta_hdr;
+#elif defined(__FreeBSD__)
+	/* We do not support HDCP in drm-kmod yet */
+	int err = 0;
+#endif
```

Comment:
Why not just do:
```c
#ifdef __linux__
    char fw_name[30];
    const struct ta_firmware_header_v1_0 *ta_hdr;
#endif
    /* We do not support HDCP in drm-kmod yet */
    int err = 0;
```

Reply:
Tried to do as minimal changes as I could. Not to rearrange lines